IPlug/IPlug_include_in_plug_src.h

```diff
@@ -270,6 +270,112 @@
     
     return 0;
   }
+
+#pragma mark - CLAP
+#elif defined CLAP_API
+
+// Make sure optional fields are defined
+
+#ifndef CLAP_MANUAL_URL
+#define CLAP_MANUAL_URL ""
+#endif
+#ifndef CLAP_SUPPORT_URL
+#define CLAP_SUPPORT_URL ""
+#endif
+#ifndef CLAP_DESCRIPTION
+#define CLAP_DESCRIPTION ""
+#endif
+#ifndef CLAP_FEATURES
+  #if PLUG_TYPE==0
+  #define CLAP_FEATURES CLAP_PLUGIN_FEATURE_AUDIO_EFFECT
+  #elif PLUG_TYPE==1
+  #define CLAP_FEATURES CLAP_PLUGIN_FEATURE_INSTRUMENT
+  #elif PLUG_TYPE==2
+  #define CLAP_FEATURES CLAP_PLUGIN_FEATURE_NOTE_EFFECT
+  #endif
+#endif
+
+std::string gPluginPath;
```

Comment:
Are these globals going to be OK with multiple instances?

Reply:
These globals don't relate to an instance - they are to do with entry to the shared library. In theory a host is allowed to load and unload without exact pairings, however, so I can take a look.

Reply:
OK, seems to be fine